draft-iab-wire-image.md

```diff
@@ -61,53 +61,69 @@ discuss in this document. It is the wire image, not the protocol's
 specification, that determines how third parties on the network paths among
 protocol participants will interact with that protocol.
 
-Several documents currently under discussion in IETF working groups and the
-IETF in general, for example
-{{?QUIC-MANAGEABILITY=I-D.ietf-quic-manageability}},
-{{?EFFECT-ENCRYPT=I-D.mm-wg-effect-encrypt}}, and
-{{?TRANSPORT-ENCRYPT=I-D.fairhurst-tsvwg-transport-encrypt}}, discuss in
-part impacts on the third-party use of wire images caused by a migration from
-protocols whose wire images are largely not confidentiality protected (e.g.
-HTTP over TCP) to protocols whose wire images are confidentiality protected
-(e.g. H2 over QUIC).
-
-This document presents the wire image abstraction with the hope that it can
-shed some light on these discussions.
+The increasing deployment of transport-layer security {{?RFC8226}} to protect
+application-layer headers and payload, as well as the definition and deployment
+of QUIC {{?I-D.ietf-quic-transport}}, a transport protocol which encrypts most
+of its own control information, bring new relevance to this question. QUIC is,
+in effect, the first IETF-defined transport protocol to take care of the
+minimization of its own wire image, for reasons of ossification prevention as
+well as reducing information radiation off the protocol for end-to-end privacy.
+
+The flipside of this trend is the impact of a less visible wire image on various
+functions driven by third-party observation of the wire image. {{?RFC8404}}
+examines this issue from a network operator's viewpoint, and
+{{?I-D.ietf-tsvwg-transport-encrypt}} focuses on transport-layer implications of
+increasing encryption. {{?I-D.ietf-quic-manageability}} is, in part, a
+third-party user's guide to the QUIC wire image. In contrast to those documents,
+this draft treats the wire image as a pure abstraction, with the hope that it
+can shed some light on these discussions.
 
 # Definition
 
-More formally, the wire image of a protocol consists of the sequence of
-messages sent by each participant in the protocol, each expressed as a
-sequence of bits with an associated arbitrary-precision time at which it was
-sent.
+More formally, the wire image of a protocol stack observed at a given point
+in the network consists of the sequence of packets sent by each participant in
+the communication, each expressed as a sequence of bits with an associated
```

Comment:
Nit: I think "with *the* associated... time at which it", not "with *an* associated", flows better for discussing a specific "each" packet.